editor.planx.uk/src/routes/team.tsx

```diff
@@ -90,8 +90,10 @@ const routes = compose(
 
     "/:flow/service": setFlowAndLazyLoad(() => import("./serviceSettings")),
 
-    "/:flow/submissions-log": setFlowAndLazyLoad(
-      () => import("./submissionsLog"),
+    "/submissions-log": lazy(() => import("./teamSubmissionsLog")),
+
+    "/submissions-log/:flow": setFlowAndLazyLoad(
+      () => import("./flowSubmissionsLog"),
```

Comment:
I think that if we make this the same route, but with an optional query param for a pre-set filter of flowSlug (or flowId) we might give ourselves a bit more flexibility here.

So rather than `/submissions-log/:flow` being a unique route, we just have a link at flow level that points to `/:teamSlug/submissions-log?flow=:flowSlug`.

This then take the user to a pre-filtered view of the same page, and they could remove this filter if they wish to do so. This might be a little simpler and have a lot of the conditionals we currently have on this component. This might just be something we need to do in future once we have the datagrid in place and might not be wholly practical just now. What do you think?